certbot/display/util.py

```diff
@@ -179,9 +179,12 @@ def checklist(self, message, tags, default_status=True, **unused_kwargs):
             self._print_menu(message, tags)
 
             code, ans = self.input("Select the appropriate numbers separated "
-                                   "by commas and/or spaces")
+                                   "by commas and/or spaces, or press enter to "
+                                   "select all domains shown")
```

Comment:
We probably shouldn't say "domains" here as this is supposed to be a general method for displaying a checklist. Perhaps we should change it to "options"?

Also, nit:

Despite this being largely the wording I proposed, I'm not sure I like "press enter to select all". You also have to press enter to select a specific options after entering the desired numbers. While this is probably unlikely to confuse people, perhaps we should say "simply/just press enter" or something more verbose like "press enter without selecting any numbers to select all".